decidim-core/app/helpers/decidim/sanitize_helper.rb

```diff
@@ -112,9 +112,10 @@ def content_handle_locale(body, all_locales, extras, links, strip_tags)
     #
     # @return ActiveSupport::SafeBuffer
     def render_sanitized_content(resource, method)
-      content = present(resource).send(method, links: true, strip_tags: !safe_content?)
+      content = present(resource).send(method, links: true, strip_tags: !try(:safe_content?))
 
-      return decidim_sanitize(content, {}) unless safe_content?
+      return decidim_sanitize(content, {}) unless try(:safe_content?)
+      return decidim_sanitize_editor_admin(content, {}) if try(:safe_content_admin?)
```

Comment:
@ahukkanen I am review-ing this PR, and i can see the debates field regarding ("Information updates" and "Instructions to participate") do not support the iframes (like adding the video). 

Shall we consider those fields as well for this change? or is it something that i would handle with #10130?

Reply:
You already printed the information in : https://github.com/decidim/decidim/issues/10130#issuecomment-1451821548 Closing the comment as invalid.